pkg/virt-launcher/virtwrap/api/schema.go

```diff
@@ -963,7 +963,7 @@ type VideoModel struct {
 }
 
 type Graphics struct {
-	AutoPort      string          `xml:"autoPort,attr,omitempty"`
+	AutoPort      string          `xml:"autoport,attr,omitempty"`
```

Comment:
In your commit message, would you explain why this is a must? What behavior is currently broken? And can it be automatically tested?

Reply:
> In your commit message, would you explain why this is a must? What behavior is currently broken? And can it be automatically tested?

We can find graphics xml format at:
https://libvirt.org/formatdomain.html#graphical-framebuffers
the autoport param must be all lowercase.

Reply:
Thanks. Please state these details in the commit message.

I am not a maintainer of this repository. However, if your company is willing to add tests for spice and own the code, you can propose a PR. By owning the code, I mean that if it breaks, you are responsible to fixing it. If you do not, the code would be dropped.

Also, you are welcome to add your company to https://github.com/kubevirt/kubevirt/blob/main/ADOPTERS.md

Reply:
How to propose a PR?